.github/workflows/main.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+    - name: Prepare
+      run: |
+        sudo apt-get update
+        sudo apt-get install libefiboot-dev check grep ninja-build meson gcc-5 libblkid-dev lcov valgrind gnu-efi libefivar-dev python3-pip ruby ruby-dev rubygems python3-setuptools
+        sudo gem install coveralls-lcov
```

Comment:
Is this still usable? I'm not sure how coveralls is working in this new world order.

Reply:
yes, still working and it's required by the test suite - I can check later if we can get rid of it or not.

Reply:
Ah seems fine, looking at the output.